app/services/podcasts/get_media_url.rb

```diff
@@ -35,7 +35,7 @@ def result_struct
     def url_reachable?(url)
       url = Addressable::URI.parse(url).normalize.to_s
       HTTParty.head(url).code == 200
-    rescue Net::OpenTimeout, SystemCallError, URI::InvalidURIError, Addressable::URI::InvalidURIError
+    rescue Net::OpenTimeout, SystemCallError, URI::InvalidURIError, Addressable::URI::InvalidURIError, SocketError
```

Comment:
I think if this list is this long, we might want to just name it 

```ruby
HANDLED_ERRORS = [
  Net::OpenTimeout,
  SystemCallError,
  URI::InvalidURIError,
  Addressable::URI::InvalidURIError,
  SocketError
  ].freeze
```
  
```ruby
  rescue HANDLED_ERRORS # can't remember if you want a *splat here
     false
```

Reply:
Thank you for this suggestion!